php-ts-face.el

```diff
@@ -0,0 +1,44 @@
+;;; php-ts-face.el --- Face definitions for PHP script  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2023  Friends of Emacs-PHP development
+
+;; Author: USAMI Kenta <tadsan@zonu.me>
+;; Created: 5 May 2019
+;; Version: 1.25.0
+;; Keywords: faces, php
+;; Homepage: https://github.com/emacs-php/php-mode
+;; License: GPL-3.0-or-later
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; Face definitions for PHP script.
+
+;;; Code:
+
+;;;###autoload
+
+(defface php-this '((t (:inherit php-constant)))
```

Comment:
I'm not sure if we need this file. We already have a face for $this here:
https://github.com/emacs-php/php-ts-mode/blob/master/php-face.el#L159-L167

Reply:
If `php-mode` was loaded before, file `php-face.el` from `php-ts-mode.el` is not evaluated

Reply:
$ symbol is not allowed in capture-name's in queries, so using original faces from `php-mode` is not possible.

Reply:
Maybe we should just rename them. What do you think @zonuexe?

Reply:
@craynot @piotrkwiecinski 
Sorry for the late reply.

I have [renamed them](https://github.com/emacs-php/php-ts-mode/pull/68) to face names without the `$`, so please run `git rebase master`. The `php-ts-face.el` files should no longer be needed.

Reply:
@zonuexe thank you! 
`php-ts-face.el` was removed